### PR TITLE
[#155] Describe how to configure helm-lsp for workspace symbols

### LINKED ISSUE #155: Document usage of workspace symbols for Emacs users

### PR DESCRIPTION
**Pull request by jfacorro**
### Description

Introduce sections in Emacs configuration.

Fixes #155.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,17 +69,16 @@ file (assuming the `erlang_ls` executable is available in your `PATH`:
...
-    ;; TCP connection timeout
-    (setq lsp-tcp-connection-timeout 500)
-    ;; Set path to erlang_ls escript (unless it's in your path)
-    (setq lsp-erlang-server-path "/path/to/erlang_ls")
+    ;; ----- company-lsp -----
+    ;; Enables better integration with company (auto-completion)
+    (package-install 'company-lsp)
+    (push 'company-lsp company-backends)
 
-    ;; Override the key bindings from erlang-mode to use LSP for completion
```
Comment (robertoaloi) [inline README.md]: @jfacorro Why did you delete this? I needed it for my completion to work correctly (I also have the other parts you are adding).

Comment (robertoaloi) [reply README.md]: Restoring in #276

**Comment by jfacorro** [comment]
Theas are the instructions in the `company-lsp`. Could the previous ones be something specific to your Emacs configuration?

**Comment by robertoaloi** [comment]
The problem is that the keybinding is set by `erlang-mode` itself, so no wonder this is not part of the generic `company-lsp` instructions. But many users of erlang_ls will have that mode enabled.
